packages/http-client/src/__tests__/ceramic-http-client.test.ts

```diff
@@ -15,11 +30,81 @@ const initial = {
   ],
 } as unknown as StreamState
 
+const GET_RESPONSE = {
+  ...initial,
+  state: initial,
+}
+
+let did: DID
+let getDidFn
+let client: CeramicClient
+
+beforeEach(() => {
+  jest.resetAllMocks()
+})
+
 describe('buildStreamFromState', () => {
   test('build instance of Streamtype', async () => {
-    const client = new CeramicClient(API_URL)
+    client = new CeramicClient(API_URL)
     const a = client.buildStreamFromState(initial)
     expect(a).toBeInstanceOf(TileDocument)
     expect(a.id.cid).toEqual(FAKE_CID_1)
   })
 })
+
+describe('URL constructor', () => {
+  beforeAll(async () => {
+    const seed = randomBytes(32)
+    const provider = new Ed25519Provider(seed)
+    const actingDid = new DID({ provider, resolver: KeyResolver.getResolver() })
+    await actingDid.authenticate()
+    did = actingDid
+    getDidFn = () => {
```

Comment:
`did` is never used in any of the tests, and `getDidFn` is only used in one test, the `setDID` test.  Instead of putting this into the test fixture setup, could you put this into a helper function that just returns a DID and use that function only in the `setDID` test?